packages/aila/src/core/chat/AilaChat.ts

```diff
@@ -40,6 +40,8 @@ export class AilaChat implements AilaChatService {
   private readonly _patchEnqueuer: PatchEnqueuer;
   private readonly _llmService: LLMService;
   private readonly _promptBuilder: AilaPromptBuilder;
+  private _iteration: number | undefined;
```

Comment:
We now have an "iteration" number on the chat which increments each time we save it